templates/client-auth-n.cfg.xml.erb

```diff
@@ -14,13 +14,21 @@
   <%- end -%>
 <%- end -%>
 <%- if @ignore_tenant_roles -%>
-    <ignore-tenant-roles>
+        <ignore-tenant-roles>
```

Comment:
Because the current resulting xml is like this:
```
<?xml version="1.0" encoding="UTF-8"?>

<client-auth xmlns="http://docs.openrepose.org/repose/client-auth/v1.0">
    <openstack-auth  tenanted="false" group-cache-timeout="60000" connectionPoolId="client-auth-pool" token-cache-timeout="600000"  send-all-tenant-ids="true" xmlns="http://docs.openrepose.org/repose/client-auth/os-ids-auth/v1.0">
        <identity-service username="***masked****" password="***masked***" uri="https://some.identity.url/v2.0" />
    <ignore-tenant-roles>
          <role>identity:admin</role>
    </ignore-tenant-roles>
    </openstack-auth>
</client-auth>
```